### DISCUSSION TITLE
Swagger的语言修订

### DISCUSSION THREAD

**Discussion by zhongshijie1995** [fastapi/fastapi#8468] [Questions] [answered]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
any
```

### Description

<img width="631" alt="image" src="https://user-images.githubusercontent.com/29966384/164293304-187c8b5d-f2b2-4b64-aeb3-7064078a0609.png">
类似上图，甚至是整个Swagger文档中的非API部分的界面元素，可否通过FastAPI中类似指定其为中文？

### Operating System

Linux, Windows, macOS, Other

### FastAPI Version

0.75.1

### Python Version

3.6

**Comment by yzqdev** [comment, marked as answer]
The document is here [https://fastapi.tiangolo.com/zh/tutorial/metadata/](https://fastapi.tiangolo.com/zh/tutorial/metadata/), you could change some metadata.

If you want to know more definitions about swagger, you could refer to [swagger-ui](https://github.com/swagger-api/swagger-ui) and have a look at their documents. Hope this may help you.
